other-tabs/collaboration/distribute-tasks-with-inboxes.md

```diff
@@ -0,0 +1,97 @@
+
+In the "Todo" section of the inbox, an operator can click on the "Start processing button" to have a task automatically assigned, and being redirected to the record.
+
+The operator then does what is necessary to achieve the current workflow, for example trigger a `Validate Document` action. Then the operator can click on the "Next ticket" button on the top of Forest Admin UI, to be assigned to a new record.
+
+### Doing section
+
+On the "Doing" section, the operator can view the tasks that are still in the segment and that are currently assigned to he or she.
+
+### Backlog section
+
+This section is accessible for operators with a Manager, Editor or Administrator permission level.
+
+It provides an overview of the current inbox.
+
+You can see all the records in the segment with their current task state, and which operator is assigned to it.
+
+For the "Doing" tasks, you can see the date when the task was started.
+
+For the "Completed" tasks, you can also see the time it took for the task to get out of the segment. If the records gets out of the segment using an action requiring an approval, it includes the approval time.
+
+### Manual task assignment
+
+Assigning a task to an operator, makes this record be the next one to be assigned for this operator, bypassing the sorting rules of the inbox.
+
+It is possible to manually assign a task to a specific operator from the [backlog](#backlog-section).
+
+It is also possible to manually assign a task to an operator from the record view, through the native Forest Admin action "Assign to...".
+
+From this action, you can assign a task to operators that are not in you current team and also select the inbox. The team should have an inbox based on a segment matching the record state.
+
+## Inbox settings
+
+Inbox settings are accessible from the Edit layout mode, in the menu near the inbox name.
+
+You can rename and delete inboxes from the settings.
+
+### Organize your inboxes with folders
+
+Inbox folders let you organize the way your inboxes are displayed to your operators.
+In the settings, under the "Inbox folder" setting you can add the current inbox in an existing folder, or create a new folder.
+
+### Set a number of maximum concurrent tasks per operators
+
+In the settings you can configure a maximum number of tasks that can be assign at the same time for an operator.
+
+When this number is reached for an operator, when trying clicking on "Next ticket" he or she will be redirected to the doing tasks.
+
+Tasks manually assigned from the backlog or a record ignore this behavior.
+
+{% hint style="info" %}
+This setting is disabled by default
+{% endhint %}
+
+### Automatically unassign tasks
+
+You can configure a maximum time before automatically unassigning a task, making the record available for other operators.
```

Comment:
You should specify it's based on the creation date on not the start date (only impact manual assignment)